core/email.py

```diff
@@ -141,7 +145,7 @@ def sendEMail(*,
               headers: Dict[str, str] = None,
               attachments: List[Dict[str, Any]] = None,
```

Comment:
`attachments` now can be a `Callable[[], List]]` as well?

Reply:
Yes, this is a side effect of the extension of the `normalize_to_list` method. 
But I didn't want to over-engineer this and make the callable option parameter-controlled. 
However, since I don't see the point in passing attachments as a method, I didn't document it further, but just left it as an "Easter-Egg".

For me we can extend the method signature, but for me it would be ok to leave it as it is, because this should not be the primary data type.